crates/telio-nurse/src/qos.rs

```diff
@@ -171,6 +172,8 @@ pub struct Analytics {
     ping_channel_rx: mpsc::Receiver<(PublicKey, DualPingResults)>,
     ping_channel_tx: mpsc::WeakSender<(PublicKey, DualPingResults)>,
     buckets: u32,
+    #[cfg(test)]
+    ping_cnt: u32,
```

Comment:
Maybe it would make pore sence to create a TestPing ping_backend?

Reply:
Yeah, it would make sense. But maybe I'll fill a separate task to do that, because this will touch all the tests (we'll need mocked latency to nodes).

Reply:
+1